From 68a40c88f5294c58a90292c04e982226094f1603 Mon Sep 17 00:00:00 2001
From: t <tristan75015@gmail.com>
Date: Tue, 13 Oct 2026 17:48:34 +0200
Subject: [PATCH] Documentation: statistics of the rotations

The rotations leave the statistics unchanged only if their time step scaling factor is not greater than 1: the stlpar and openmp backends set it to 1 otherwise (as said by the comment of TFEL copied here).
---
 include/MGIS/Behaviour/BatchDataView.hxx | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/MGIS/Behaviour/BatchDataView.hxx b/include/MGIS/Behaviour/BatchDataView.hxx
index 95643cb..18980ce 100644
--- a/include/MGIS/Behaviour/BatchDataView.hxx
+++ b/include/MGIS/Behaviour/BatchDataView.hxx
@@ -237,7 +237,9 @@ typedef struct mgis_bv_BatchRotationDataView {
   mgis_bv_ConstField rotation_matrices;
   /*!
    * \brief options of the execution of the computations. The rotations never
-   * fail: the statistics, if given, are not modified.
+   * fail: the statistics, if given, are not modified if their time step
+   * scaling factor is not greater than 1 (the `stlpar` and `openmp` backends
+   * set it to 1 otherwise).
    */
   mgis_bv_BatchExecutionOptions execution;
 } mgis_bv_BatchRotationDataView;
-- 
2.43.0

